### PR TITLE
fix: [CVS-9232]: clear filters and add - when there is no data for se…

### PR DESCRIPTION
**Pull request by vasuki-kalluri**
…rvice health score

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by vasuki-kalluri** [comment]
trigger cypress

**Comment by vasuki-kalluri** [comment]
image
image

File: src/modules/85-cv/pages/changes/CVChanges.tsx [hunk 1]
```diff
@@ -81,6 +89,16 @@ export const CVChanges = ({ updateTime }: { updateTime?: Date }): JSX.Element =>
     return getStartAndEndTime((selectedTimePeriod?.value as string) || '').endTimeRoundedOffToNearest30min
   }, [selectedTimePeriod, serviceOptions, environmentOptions, sourceTypes, connectorOptions])
 
+  const onClickHandler = () => {
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/changes/CVChanges.tsx:92]: rename function to meaningful name

**Comment by deepesh-ui** [comment]
Please add a test to verify clear filter functionality both in JEST and Cypress

File: src/modules/85-cv/pages/changes/CVChanges.tsx (hunk 1 above)
Comment (vasuki-kalluri) [reply src/modules/85-cv/pages/changes/CVChanges.tsx]: Done

File: src/modules/85-cv/pages/changes/__tests__/CVChanges.test.tsx [hunk 2]
```diff
@@ -278,6 +278,36 @@ describe('Unit tests for CVChanges', () => {
     expect(mockFetch).toHaveBeenCalledTimes(3)
   })
 
+  test.only('clear Filter', async () => {
```
Comment (anjanBalgovind) [inline src/modules/85-cv/pages/changes/__tests__/CVChanges.test.tsx:281]: remove the only here

Comment (vasuki-kalluri) [reply src/modules/85-cv/pages/changes/__tests__/CVChanges.test.tsx:281]: ooops :)

**Review by anjanBalgovind — APPROVED** [review]

**Review by mtran7 — APPROVED** [review]

**Pull request merged**